### PR TITLE
[17.0][MIG] partner_aging: Migration to 17.0 

### PR DESCRIPTION
**Pull request by miquelalzanillas**
Module migrated to version 17.0

cc https://github.com/APSL 161354

@javierobcn @mpascuall @BernatObrador @ppyczko

 * [Issue] [ #683 ]

### CODE REVIEW

File: partner_aging/wizard/res_partner_aging_customer.xml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+                create="false"
+                delete="false"
+                decoration-danger="max_days_overdue &gt; 0 and total &gt; 0"
+            >
+                <field name="partner_id" string="Customer" />
+                <field name="salesman" />
+                <field name="date_due" string="Invoice Date" />
+                <field name="inv_date_due" string="Due Date" />
+                <field name="max_days_overdue" />
+                <field name="invoice_id" invisible="1" />
```
Comment (ppyczko) [inline partner_aging/wizard/res_partner_aging_customer.xml]: invisible="1" should be replaced with column_invisible="1" in the tree view
```suggestion
                <field name="invoice_id" column_invisible="1" />
```

File: partner_aging/readme/USAGE.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+To use this module, you need to:
+
+1.  Go to Accounting
+2.  Click on Sales \> Customer Aging or Purchases \> Supplier Aging
+3.  Change the date if necessary
```
Comment (ppyczko) [inline partner_aging/readme/USAGE.md]: Maybe we could change the path to better match the actual naming? Not detrimental. 

```suggestion
1.  Go to Invoicing
2.  Click on Customers \> Customer Aging or Vendors \> Supplier Aging
3.  Change the date if necessary
```

**Review by ppyczko — CHANGES_REQUESTED** [review]

File: partner_aging/wizard/res_partner_aging_customer.xml (hunk 1 above)
Comment (miquelalzanillas) [reply partner_aging/wizard/res_partner_aging_customer.xml]: Thanks @ppyczko ! Fixed.

File: partner_aging/readme/USAGE.md (hunk 2 above)
Comment (miquelalzanillas) [reply partner_aging/readme/USAGE.md]: Fixed.

**Review by ppyczko — APPROVED** [review]
LGTM! Tested in runboat

**Review by miquelpascual — APPROVED** [review]
LGTM, tested in runboat

File: partner_aging/wizard/res_partner_aging_customer.py [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+    not_due = fields.Float("Not Due Yet", readonly=True)
+    days_due_01to30 = fields.Float("1/30", readonly=True)
+    days_due_31to60 = fields.Float("31/60", readonly=True)
+    days_due_61to90 = fields.Float("61/90", readonly=True)
+    days_due_91to120 = fields.Float("91/120", readonly=True)
+    days_due_121togr = fields.Float("+121", readonly=True)
+    max_days_overdue = fields.Integer("Days Outstanding", readonly=True)
+    invoice_ref = fields.Char("Our Invoice", size=25, readonly=True)
+    invoice_id = fields.Many2one("account.move", "Invoice", readonly=True)
+    salesman = fields.Many2one("res.users", "Sales Rep", readonly=True)
```
Comment (peluko00) [inline partner_aging/wizard/res_partner_aging_customer.py:28]: Remove readonly attributes and put it in view please

File: partner_aging/wizard/res_partner_aging_supplier.py [hunk 4]
```diff
@@ -0,0 +1,367 @@
...
+    total = fields.Float(readonly=True)
+    not_due = fields.Float("Not Due Yet", readonly=True)
+    days_due_01to30 = fields.Float("1/30", readonly=True)
+    days_due_31to60 = fields.Float("31/60", readonly=True)
+    days_due_61to90 = fields.Float("61/90", readonly=True)
+    days_due_91to120 = fields.Float("91/120", readonly=True)
+    days_due_121togr = fields.Float("+121", readonly=True)
+    invoice_ref = fields.Char("Their Invoice", size=25, readonly=True)
+    invoice_id = fields.Many2one("account.move", "Invoice", readonly=True)
+    salesman = fields.Many2one("res.users", "Sales Rep", readonly=True)
```
Comment (peluko00) [inline partner_aging/wizard/res_partner_aging_supplier.py:28]: Here the same

**Review by peluko00 — CHANGES_REQUESTED** [review]

File: partner_aging/wizard/res_partner_aging_customer.py (hunk 3 above)
Comment (miquelalzanillas) [reply partner_aging/wizard/res_partner_aging_customer.py:28]: I'm not sure about that. Can you please explain why?

Comment (peluko00) [reply partner_aging/wizard/res_partner_aging_customer.py:28]: Here you will see about it
image

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0

Comment (miquelalzanillas) [reply partner_aging/wizard/res_partner_aging_customer.py:28]: Thanks for the explanation. The data in this fields doesn't have to be imported in any case. So from my side the actual code is correct.

File: partner_aging/wizard/res_partner_aging_supplier.py (hunk 4 above)
Comment (miquelalzanillas) [reply partner_aging/wizard/res_partner_aging_supplier.py:28]: The data in this fields doesn't have to be imported in any case. So from my side the actual code is correct.

**Review by peluko00 — APPROVED** [review]
LGTM

**Comment by miquelalzanillas** [comment]
Hi @HaraldPanten , 

can you do a check here and see if can be merged? 

Thanks!

**Review by HaraldPanten — APPROVED** [review]
I'm not using this module, but I will move it forward as it has several validations and your team was who carried out the migration to V16. So... you've been kind of maintainers.

**Pull request merged**